.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint and Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 2
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ github.ref }}
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+      - name: npm install
+        run: npm ci --only=prod
```

Comment:
I suspect subsequent steps will fail because `eslint` and `mocha` won't be installed: https://github.com/Brightspace/d2l-license-checker/blob/b3ecf7b16ba3543c78603f6daa51b4bb6d0af86b/package.json#L12-L13

Is there a reason you only want prod deps here?

Reply:
This was a copy paste mishap, thanks for catching that.